### PR TITLE
i18n(zh-cn): Translate `distribute/Signing/*`

### PR DESCRIPTION
**Pull request by GuoJikun**
#### Description

Includes file.

- [x]  `distribute/Signing/android.mdx`
- [x]  `distribute/Signing/ios.mdx`
- [x]  `distribute/Signing/linux.mdx`
- [x]  `distribute/Signing/macos.mdx`
- [x]  `distribute/Signing/windows.mdx`

### CODE REVIEW

File: src/content/docs/zh-cn/distribute/Signing/ios.mdx [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+title: iOS 代码签名
+sidebar:
+  label: iOS
+  order: 4
+---
+
+在 iOS 上进行代码签名是必需的，以便通过官方[苹果应用商店] 或可能的欧盟替代市场分发您的应用程序，并且一般来说可以安装和执行在最终用户设备上。
```
Comment (dklassic) [inline src/content/docs/zh-cn/distribute/Signing/ios.mdx]:
```suggestion
在 iOS 上进行代码签名是必需的，以便通过官方[苹果应用商店]或可能的欧盟替代市场分发您的应用程序，并且一般来说可以安装和执行在最终用户设备上。
```
Formatting nitpick

File: src/content/docs/zh-cn/distribute/Signing/ios.mdx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+你还需要下载私钥，它只能下载一次，并且只有在页面重新加载后才可见(该按钮显示在新创建密钥的表格行上)。
+私钥文件路径必须通过 `APPLE_API_KEY_PATH` 环境变量设置。
+
+## 手动签名
+
+要手动为你的 iOS 应用签名，你可以通过环境变量提供证书和移动配置文件:
+
+- **IOS_CERTIFICATE**: 从 Keychain 导出的证书的 base64 表示形式。
+- **IOS_CERTIFICATE_PASSWORD**: 从 Keychain 导出时设置的证书密码。
+- **IOS_MOBILE_PROVISION**: 授权描述文件的 Base64 表示。
```
Comment (dklassic) [inline src/content/docs/zh-cn/distribute/Signing/ios.mdx]:
```suggestion
要手动为你的 iOS 应用签名，你可以通过环境变量提供证书和移动配置文件：

- **IOS_CERTIFICATE**：从 Keychain 导出的证书的 base64 表示形式。
- **IOS_CERTIFICATE_PASSWORD**：从 Keychain 导出时设置的证书密码。
- **IOS_MOBILE_PROVISION**：授权描述文件的 Base64 表示。
```
Formatting nitpick

File: src/content/docs/zh-cn/distribute/Signing/ios.mdx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+保存并作为 `IOS_CERTIFICATE` 环境变量的值使用。
+
+需要设置 `IOS_CERTIFICATE_PASSWORD` 变量的值必须为证书密码。
+
+:::tip[选择证书类型]
+每个导出方法都必须使用适当的证书类型。
+
+- **debugging**: 苹果开发或 iOS 应用开发
+- **app-store-connect**: 苹果发行版或 iOS 发行版 (App Store Connect 和 Ad Hoc)
+- **ad-hoc**: 苹果发行版或 iOS 发行版 (App Store Connect 和 Ad Hoc)
```
Comment (dklassic) [inline src/content/docs/zh-cn/distribute/Signing/ios.mdx]:
```suggestion
- **debugging**：苹果开发或 iOS 应用开发
- **app-store-connect**：苹果发行版或 iOS 发行版 (App Store Connect 和 Ad Hoc)
- **ad-hoc**：苹果发行版或 iOS 发行版 (App Store Connect 和 Ad Hoc)
```
Formatting nitpick

File: src/content/docs/zh-cn/distribute/Signing/ios.mdx [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+创建配置文件后，下载它并运行以下 `base64` 命令来转换该配置文件并将其复制到剪贴板：
+
+```
+base64 -i <path-to-profile.mobileprovision> | pbcopy
+```
+
+剪贴板中的值现在是配置描述文件的 base64 表示。
+保存它并将其用作 `IOS_MOBILE_PROVISION` 环境变量值。
+
+现在你可以构建你的 iOS 应用程序并在 App Store 上发布了!
```
Comment (dklassic) [inline src/content/docs/zh-cn/distribute/Signing/ios.mdx]:
```suggestion
现在你可以构建你的 iOS 应用程序并在 App Store 上发布了！
```
Formatting nitpick

File: src/content/docs/zh-cn/distribute/Signing/linux.mdx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+您应该特别注意将私钥和公钥备份到安全的位置。
+
+### 签名
+
+可以通过设置以下环境变量在 AppImage 中嵌入签名：
+
+- **SIGN**: 设置为 `1` 为 AppImage 签名。
+- **SIGN_KEY**: 可选变量，用于使用特定的 GPG 密钥 ID 进行签名。
+- **APPIMAGETOOL_SIGN_PASSPHRASE**: 签名密钥密码。如果未设置，gpg 将显示一个对话框，以便您可以输入它。在构建 CI/CD 平台时必须设置此参数。
+- **APPIMAGETOOL_FORCE_SIGN**: 默认情况下，即使签名失败，也会生成 AppImage。要在发生错误时退出，你可以将此变量设置为 `1`。
```
Comment (dklassic) [inline src/content/docs/zh-cn/distribute/Signing/linux.mdx]:
```suggestion
- **SIGN**：设置为 `1` 为 AppImage 签名。
- **SIGN_KEY**：可选变量，用于使用特定的 GPG 密钥 ID 进行签名。
- **APPIMAGETOOL_SIGN_PASSPHRASE**：签名密钥密码。如果未设置，gpg 将显示一个对话框，以便您可以输入它。在构建 CI/CD 平台时必须设置此参数。
- **APPIMAGETOOL_FORCE_SIGN**：默认情况下，即使签名失败，也会生成 AppImage。要在发生错误时退出，你可以将此变量设置为 `1`。
```
Formatting nitpick

File: src/content/docs/zh-cn/distribute/Signing/windows.mdx [hunk 6]
```diff
@@ -0,0 +1,284 @@
...
+1. 我们需要在工作流中添加一个步骤来将证书导入 Windows 环境。该工作流完成以下工作。
+
+   1. 将 GitHub 密钥分配给环境变量
+   2. 创建一个新的 `certificate` 目录
+   3. 将 `WINDOWS_CERTIFICATE` 导入到 tempCert.txt 文件中
+   4. 使用 `certutil` 将 tempCert.txt 从 base64 解码为 `.pfx` 文件。
+   5. 移除 tempCert.txt 文件
+   6. 将 `.pfx` 文件导入到 Windows 的证书存储中，并将 `WINDOWS_CERTIFICATE_PASSWORD` 转换为安全字符串，以便在导入命令中使用。
+
+2. 我们将使用 [`tauri-action` 发布模板].
```
Comment (dklassic) [inline src/content/docs/zh-cn/distribute/Signing/windows.mdx]:
```suggestion
2. 我们将使用 [`tauri-action` 发布模板]。
```
Formatting nitpick

File: src/content/docs/zh-cn/distribute/Signing/windows.mdx [hunk 7]
```diff
@@ -0,0 +1,284 @@
...
+在 Azure portal 的登录页面中，转到 “Microsoft Entra ID” 服务，并转到 “Manage > App registrations” 页面。
+点击 “New registration” 创建新应用。创建应用后，您将重定向到应用详情页面，在那里您可以看到“应用（客户端）ID” 和“目录（租户）ID” 值。
+将这些 ID 分别设置为 `AZURE_VAULT_ID` 和 `AZURE_TENANT_ID` 环境变量。
+
+在 “Manage > Certificates & secrets” 页面单击 “New client secret” 按钮，并将 “Value” 列中的文本设置为 “AZURE_CLIENT_SECRET” 环境变量的值。
+
+设置好所有凭据后，回到您的密钥库页面并导航到“访问控制（IAM）”页面。
+您必须为新创建的应用程序分配 “Key Vault Certificate User” 和 “Key Vault Crypto User” 角色。
+
+设置好所有这些变量后，运行 `tauri build` 将生成签名的Windows安装程序!
```
Comment (dklassic) [inline src/content/docs/zh-cn/distribute/Signing/windows.mdx]:
```suggestion
设置好所有这些变量后，运行 `tauri build` 将生成签名的Windows安装程序！
```
Formatting nitpick

**Review by dklassic — CHANGES_REQUESTED** [review]
Some nitpicks here and there and should be good to go!

File: src/content/docs/zh-cn/distribute/Signing/ios.mdx [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+## 自动签名
+
+让 Xcode 管理应用的签名和配置是将 iOS 应用导出以进行分发的最方便的方式。
+它会自动注册您的包标识符，处理 iOS 功能变化，并根据您选择的导出方法配置适当的证书。
+
+默认情况下启用了自动签名，并且在本地机器上使用时会使用 Xcode 中配置的账户进行身份验证。\
+要注册您的帐户，请打开 Xcode 应用程序，然后在 `Xcode > Settings` 菜单中打开设置页面，切换到账户选项卡，并点击 `+` 图标。
+
+要使用 CI/CD 平台上的自动签名功能，您必须创建一个 App Store Connect API 密钥，并定义 `APPLE_API_ISSUER`、`APPLE_API_KEY` 和 `APPLE_API_KEY_PATH` 环境变量。
+打开 [App Store Connect 用户和访问页面]，选择 Keys 选项卡，单击 Add 按钮并选择名称和开发者访问权限。
```
Comment (GuoJikun) [inline src/content/docs/zh-cn/distribute/Signing/ios.mdx]:
```suggestion
打开 [App Store Connect 用户和访问页面]，选择 Integrations 选项卡，单击 Add 按钮并选择名称和管理员访问权限。
```

File: src/content/docs/zh-cn/distribute/Signing/macos.mdx [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+5. 将 `certificate-base64.txt` 文件的内容设置为 `APPLE_CERTIFICATE` 环境变量的值。
+6. 将证书密码设置为 `APPLE_CERTIFICATE_PASSWORD` 环境变量。
+
+## 公正
+
+要公证您的应用程序，您必须为 Tauri 提供凭证，以便与 Apple 进行认证：
+
+- APPLE_API_ISSUER、APPLE_API_KEY 和 APPLE_API_KEY_PATH：使用 App Store Connect API 密钥进行身份验证
+
+  打开 [App Store Connect 用户和访问页面]，选择 Keys 选项卡，单击 Add 按钮并选择名称和开发者权限。
```
Comment (GuoJikun) [inline src/content/docs/zh-cn/distribute/Signing/macos.mdx]:
```suggestion
  打开 [App Store Connect 用户和访问页面]，选择 Integrations 选项卡，单击 Add 按钮并选择名称和开发者权限。
```

Comment (GuoJikun) [reply src/content/docs/zh-cn/distribute/Signing/macos.mdx]: related: [Integrations](https://github.com/tauri-apps/tauri-docs/commit/97fa2051c7e8003cbd726c6668ad6a7c50dfaf4d)

File: src/content/docs/zh-cn/distribute/Signing/ios.mdx (hunk 8 above)
Comment (GuoJikun) [reply src/content/docs/zh-cn/distribute/Signing/ios.mdx]: related: [Integrations](https://github.com/tauri-apps/tauri-docs/commit/97fa2051c7e8003cbd726c6668ad6a7c50dfaf4d)

**Review by dklassic — APPROVED** [review]

**Pull request merged**